pages/words/[id].js

```diff
@@ -35,43 +53,60 @@ export default function SingleWordPage() {
     );
     setIsShowMode(true);
   }
+  function handleDeleteEntry() {
+    setToast("enter");
+    setTranslationList(
+      translationList.filter((translation) => translation.id !== id)
+    );
+    setTimeout(exitToast, 2000);
+  }
+
+  function exitToast() {
+    setToast("exit");
+  }
 
   return (
     <main>
       <>
         <h1>Word entry {!isShowMode && ": edit"}</h1>
-
+        <ToastMessage toast={toast} />
         {isShowMode ? (
           <EditButton onClick={() => setIsShowMode(false)} />
         ) : (
           <CancelEditButton onClick={() => setIsShowMode(true)} />
         )}
         {entry && (
-          <SingleEntry
-            isFavorite={entry.isFavorite}
-            word={entry.word}
-            language={entry.language}
-            translated={entry.translated}
-            notes={entry.notes}
-            onDeleteEntry={() => {
-              router.push("/words");
-              setTranslationList(
-                translationList.filter((translation) => translation.id !== id)
-              );
-            }}
-            onToggleFavorite={() =>
-              setTranslationList(
-                translationList.map((translation) =>
-                  translation.id === id
-                    ? {
-                        ...translation,
-                        isFavorite: !translation.isFavorite,
-                      }
-                    : translation
+          <>
+            <SpeechSynthesisModule
+              word={entry.word}
+              selectedVoice={selectedVoice}
+              availableVoices={availableVoices}
+              onSubmit={(event) => handleSelectSubmit(event)}
+            />
+            <SingleEntry
+              isFavorite={entry.isFavorite}
+              word={entry.word}
+              language={entry.language}
+              translated={entry.translated}
+              notes={entry.notes}
+              voice={entry.voice}
```

Comment:
instead of passing all these single probs, you can just pass the entry and then use `entry.isFavorite`, `entry.word` etc. in the component. Then it's a bit smaller here.
```suggestion
              entry={entry}
```

Reply:
Changed it!